programmers/로또.js

```diff
@@ -0,0 +1,21 @@
+function solution(lottos, winNums) {
+    
+    let [zCnt, wCnt] = [0, 0];
+    let [best, worst] = [0, 0];
+    
+    for (let i = 0; i < 6; i++) {
+        if(lottos[i] === 0) zCnt += 1;
+        
+        for (let j = 0; j < 6; j++) {
+            if (lottos[i] === winNums[j]) wCnt += 1;
+        }
+    }
+    
+    best = Math.abs(wCnt + zCnt - 7);
```

Comment:
```suggestion
    best = 7- (wCnt + zCnt);
```
이렇게하면 메서드를 사용하지 않아도 사용할 수 있어요!